### DISCUSSION TITLE
CPU History Widget

### DISCUSSION THREAD

**Discussion by phantomate** [Lissy93/dashy#694] [Help and Support] [answered]
Hi,

I recently started to use dashy and I have a question regarding the CPU history widget. If I have dashy open, then the CPU history is tracked but if I close dashy, then it is not tracked. This leads to the fact that when I open dashy again after a few hours, the period in the history is missing. Is there a solution for this?

**Comment by lissy93** [comment, marked as answer]
No solution, as all the CPU widget does is fetch data from the API. Widgets only display live data, but don't actually store anything.

I think using something like Prometheus + Grafana would be a better fit for tracking historical CPU usage.
